docs/organizations/segments.md

```diff
@@ -0,0 +1,37 @@
+# Segments
+Segments are dimensions that Codacy reads from your provider that organizes repositories into relevant groups for you. Today, Segments is available for: [GitHub Custom Properties](#github-custom-properties)
+  
+
+### Where can Segments be utilised?
```

Comment:
I noticed you jump from header1 to header3 I think we should keep consistency and use ##